setup.py

```diff
@@ -28,7 +28,7 @@
             "black",
             "duckdb",
             "isort",
-            "mypy",
+            "mypy>=0.990",
```

Comment:
This was changed because we have the following in `serde.py`

```python
Node = t.Union[t.List["Node"], exp.DataType.Type, exp.Expression, JSON]
```

Which seems to be supported after version `0.990` of mypy ([source](https://stackoverflow.com/a/53639221)).